README.md

```diff
@@ -64,16 +64,16 @@ Docker images are available on [DockerHub][docker].
 
 1. Set the following environment variables:
 
-* `AXIOM_DEPLOYMENT_URL`: URL of the Axiom deployment to use
+* `AXIOM_URL`: **https://cloud.axiom.co**
 * `AXIOM_ACCESS_TOKEN`: **Personal Access** or **Ingest** token. Can be
```

Comment:
There's no such thing as `AXIOM_ACCESS_TOKEN`, it's called `AXIOM_TOKEN`: https://github.com/axiomhq/axiom-honeycomb-proxy/blob/main/cmd/axiom-honeycomb-proxy/main.go#L18